### DISCUSSION TITLE
where can find the project_id ?

### DISCUSSION THREAD

**Discussion by chaochaokeji** [nocodb/nocodb#1012] [Q&A] [answered]
copy project info failed

**Comment by wingkwong** [comment, marked as answer]
Copy Project Info doesn't return Project ID. 

You can find the Project ID on your URL - ``http://<host>:<port>/#/nc/<project_id>``. For example, the project id for ``http://localhost:3000/#/nc/sakila_fzr1?name=Actor&dbalias=db&type=table`` is ``sakila_fzr1``.

**Reply by chaochaokeji** [reply]
thank you
